### PR TITLE
enable use of translated activerecord column names

### PR DESCRIPTION
**Pull request by newbray**
When the class name is `UserCsv` or `Something::UserCSV`, it takes the `User` part and calls `human_attribute_name` on it to fetch the translated attribute if available for each heading in the CSV file.

`human_attribute_name` defaults to the provided string if the attribute is not present.

### CODE REVIEW

File: lib/julia/builder.rb [hunk 1]
```diff
@@ -9,13 +9,25 @@ def initialize(collection, csv_options = Hash.new)
...
       @columns ||= {}
     end
 
+    def self.model_class
+      begin
+        self.name.demodulize.sub(/Csv$/, '').constantize
+      rescue NameError
+        nil
+      end
+    end
```
Comment (bartoszkopinski) [inline lib/julia/builder.rb]: I think the syntax should be:
```ruby
def self.model_class
  name.demodulize.sub(/Csv$/, '').constantize
rescue NameError
end
```

File: lib/julia/builder.rb [hunk 2]
```diff
@@ -9,13 +9,25 @@ def initialize(collection, csv_options = Hash.new)
     end
 
     def self.column(keyname, action = nil, &block)
-      self.columns[keyname] = Action.new(keyname, action, &block)
+      if !self.model_class.nil? && self.model_class.respond_to?(:human_attribute_name)
+        translated_key = self.model_class&.human_attribute_name(keyname)
+      end
+
+      self.columns[translated_key || keyname] = Action.new(keyname, action, &block)
```
Comment (bartoszkopinski) [inline lib/julia/builder.rb]: correct syntax:
```ruby
def self.column(keyname, action = nil, &block)
  if model_class.respond_to?(:human_attribute_name)
    translated_key = model_class.human_attribute_name(keyname)
  end
 
  columns[translated_key || keyname] = Action.new(keyname, action, &block)
end   
```

Comment (bartoszkopinski) [reply lib/julia/builder.rb]: I also think `model_class` should be memoized since it's being executed twice for each column

**Comment by newbray** [comment]
thanks @bartoszkopinski updated

**Review by bartoszkopinski — APPROVED** [review]

**Pull request merged**